src/pages/Account/Create.tsx

```diff
@@ -1,84 +1,61 @@
 import React from 'react';
 import Helmet from 'react-helmet';
-import ManageAccountContainer, { ManageAccountModes } from '../../features/Account/ManageAccountForm';
+import ManageAccountForm, {
+  ManageAccountModes,
+} from '../../features/Account/ManageAccountForm';
 
 import * as CONSTANTS from '../../config/constants';
-import {
-  BackgroundImage,
-  H1,
-  MaxWidthBox,
-} from '../../shared/Elements';
-import MediaQuery from 'react-responsive';
+import { H1 } from '../../shared/Elements';
 
-import Coders from '../../assets/images/coders.svg';
+import GirlAtHome from '../../assets/images/girl-at-home.svg';
 
 const CreateAccountPage: React.FC = () => (
   <>
     <Helmet>
-      <title>
-        Create Account | {CONSTANTS.HACKATHON_NAME}
-      </title>
+      <title>Create Account | {CONSTANTS.HACKATHON_NAME}</title>
     </Helmet>
 
-    <MediaQuery maxWidth="991px">
-      <MaxWidthBox m={'auto'} paddingLeft={'50px'} paddingRight={'50px'} maxWidth={'500px'}>
-        <H1
-          fontSize={'30px'}
-          textAlign={'left'}
-          marginTop={'0px'}
-          marginBottom={'20px'}
-          marginLeft={'0px'}
-          paddingBottom={'20px'}
-          paddingTop={'70px'}
-        >
-          Your Account
-        </H1>
-        <ManageAccountContainer mode={ManageAccountModes.CREATE} />
-      </MaxWidthBox>
-    </MediaQuery>
+    <div className="SignUp--form-container">
+      <div className="SignUp--art-wrapper">
+        <img src={GirlAtHome} className="SignUp--art" alt="Background" />
+      </div>
 
-    <MediaQuery minWidth="992px">
-      <MaxWidthBox paddingLeft={'100px'} paddingRight={'50px'} maxWidth={'500px'}>
-        <H1
-          fontSize={'30px'}
-          textAlign={'left'}
-          marginTop={'0px'}
-          marginBottom={'20px'}
-          marginLeft={'0px'}
-          paddingBottom={'20px'}
-          paddingTop={'70px'}
-        >
-          Your Account
-        </H1>
-        <ManageAccountContainer mode={ManageAccountModes.CREATE} />
-      </MaxWidthBox>
-    </MediaQuery>
+      <form className="SignUp--form">
+        <H1>Create your account</H1>
+        <ManageAccountForm mode={ManageAccountModes.CREATE} />
+      </form>
 
-    <MediaQuery minWidth="992px" maxWidth="1093px">
-      <BackgroundImage
-        src={Coders}
-        top={'60px'}
-        right={'0px'}
-        imgHeight={'70%'}
-      />
-    </MediaQuery>
-    <MediaQuery minWidth="1094px" maxWidth="1199px">
-      <BackgroundImage
-        src={Coders}
-        top={'60px'}
-        right={'0px'}
-        imgHeight={'80%'}
-      />
-    </MediaQuery>
-    <MediaQuery minWidth="1200px">
-      <BackgroundImage
-        src={Coders}
-        top={'60px'}
-        right={'0px'}
-        imgHeight={'90%'}
-        position={'fixed' as 'fixed'}
-      />
-    </MediaQuery>
+      <style jsx>{`
+        /* Prefixing styles with SignUp until styled-jsx plugin integrated with babel to avoid global scoped styling conflicts */
```

Comment:
What's the issue with global scoped styling conflicts here?

Reply:
I need to setup `styled-jsx` as a babel plugin (either by ejecting from create-react-app or by using rewire), right now this css is global scoped instead of to just this component

Reply:
This seems like a P0 issue

Reply:
Fixed